dev-tools/.php-cs-fixer.php-unit-attributes.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+$config = require __DIR__.'/../.php-cs-fixer.dist.php';
+
+$config->setRules([
+    'fully_qualified_strict_types' => ['import_symbols' => true],
+    'ordered_imports' => true,
+    'php_unit_attributes' => true,
+    'phpdoc_trim' => true,
```

Comment:
does each of those needed to run under PHPUnit 12? if only some, I would keep only those. We not gonna commit them to branch yet.

Reply:
~No, I've added them for fun 😞~

If you scroll a bit to the description, click the link to the PR, open any failed job, see the logs you will see all those 3 "extra" fixers are the reason for that single test failure.

Reply:
is this what you mean?
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/13247058118/job/36976033604#step:11:38

how `phpdoc_trim` related to this failure ?

Reply:
When `php_unit_attributes` is applied to [the comment](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.68.5/tests/Console/Command/FixCommandTest.php#L26-L30):
```php
/**
 * @internal
 *
 * @covers \PhpCsFixer\Console\Command\FixCommand
 */
```
it becomes:
```php
/**
 * @internal
 *
 */
```
so `phpdoc_trim` is needed to remove that empty comment line.

Reply:
having random file with modified CS should not affect result of this test case.
That means the case is not stable , eg it should work on _fixed, constant_ fixture and not directly on codebase under development.

Reply:
I totally agree, but it is how it works now: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.68.5/tests/Console/Command/FixCommandTest.php#L176

Reply:
I'm glad you concluded to fix wrong testing setup rather than adding extra rules to migration step